Makefile

```diff
@@ -636,7 +598,7 @@ ifeq ($(IS_WINDOWS), 1)
 else
 	cp $(TARGET_DIR)/wasmer package/bin/
 ifeq ($(IS_DARWIN), 1)
-	codesign -s - package/bin/wasmer
+	# codesign -s - package/bin/wasmer || true
```

Comment:
Same, I expect this to be removed from this diff, right?

Reply:
Fixed